ui-admin/src/study/participants/participantList/ParticipantList.tsx

```diff
@@ -187,22 +192,28 @@ function ParticipantList({ studyEnvContext }: {studyEnvContext: StudyEnvContextT
       <div className="col-12">
         <LoadingSpinner isLoading={isLoading}>
```

Comment:
I suggest reviewing this with whitespace changes hidden to get a better idea of what actually changed: https://github.com/broadinstitute/juniper/pull/585/files?w=1